simplewebrtc.js

```diff
@@ -231,6 +231,7 @@ function SimpleWebRTC(opts) {
         });
     });
     this.webrtc.on('localScreenStopped', function (stream) {
+        self.emit('localScreenRemoved');
```

Comment:
@xdumaine 

Chrome shows a thin little panel at the bottom with a "Stop Sharing" button.

Does that work if they click that button?  I thought I tried that and it didn't.